<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="9" passed="5" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-06T21:13:02 EST" name="Suite" finished-at="2022-11-06T21:13:38 EST" duration-ms="36233">
    <groups>
      <group name="Smoke">
        <method signature="indexPageTest.verifyLogo()[pri:0, instance:testCases.indexPageTest@13c9d689]" name="verifyLogo" class="testCases.indexPageTest"/>
        <method signature="indexPageTest.verifyTitle()[pri:1, instance:testCases.indexPageTest@13c9d689]" name="verifyTitle" class="testCases.indexPageTest"/>
        <method signature="indexPageTest.clickOnSignInTest()[pri:2, instance:testCases.indexPageTest@13c9d689]" name="clickOnSignInTest" class="testCases.indexPageTest"/>
        <method signature="indexPageTest.verifyTitleNegative()[pri:4, instance:testCases.indexPageTest@13c9d689]" name="verifyTitleNegative" class="testCases.indexPageTest"/>
        <method signature="indexPageTest.verifyLogoNegative()[pri:5, instance:testCases.indexPageTest@13c9d689]" name="verifyLogoNegative" class="testCases.indexPageTest"/>
        <method signature="loginPageTest.doLoginTest()[pri:0, instance:testCases.loginPageTest@15de0b3c]" name="doLoginTest" class="testCases.loginPageTest"/>
        <method signature="loginPageTest.doLogoutTest()[pri:1, instance:testCases.loginPageTest@15de0b3c]" name="doLogoutTest" class="testCases.loginPageTest"/>
        <method signature="loginPageTest.doLoginTestNegative()[pri:2, instance:testCases.loginPageTest@15de0b3c]" name="doLoginTestNegative" class="testCases.loginPageTest"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2022-11-06T21:13:02 EST" name="IndexPageTest" finished-at="2022-11-06T21:13:13 EST" duration-ms="11523">
      <class name="testCases.indexPageTest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:testCases.indexPageTest@13c9d689]" started-at="2022-11-06T21:13:02 EST" name="setup" finished-at="2022-11-06T21:13:06 EST" duration-ms="4357" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLogo()[pri:0, instance:testCases.indexPageTest@13c9d689]" started-at="2022-11-06T21:13:06 EST" name="verifyLogo" finished-at="2022-11-06T21:13:06 EST" duration-ms="120" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogo -->
        <test-method signature="verifyTitle()[pri:1, instance:testCases.indexPageTest@13c9d689]" started-at="2022-11-06T21:13:06 EST" name="verifyTitle" finished-at="2022-11-06T21:13:06 EST" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitle -->
        <test-method signature="clickOnSignInTest()[pri:2, instance:testCases.indexPageTest@13c9d689]" started-at="2022-11-06T21:13:06 EST" name="clickOnSignInTest" finished-at="2022-11-06T21:13:13 EST" duration-ms="6453" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnSignInTest -->
        <test-method signature="verifyTitleNegative()[pri:4, instance:testCases.indexPageTest@13c9d689]" started-at="2022-11-06T21:13:13 EST" name="verifyTitleNegative" finished-at="2022-11-06T21:13:13 EST" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [My Store] but found [Wrong Title]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [My Store] but found [Wrong Title]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testCases.indexPageTest.verifyTitleNegative(indexPageTest.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitleNegative -->
        <test-method signature="verifyLogoNegative()[pri:5, instance:testCases.indexPageTest@13c9d689]" started-at="2022-11-06T21:13:13 EST" name="verifyLogoNegative" finished-at="2022-11-06T21:13:13 EST" duration-ms="2" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at testCases.indexPageTest.verifyLogoNegative(indexPageTest.java:81)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogoNegative -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.indexPageTest@13c9d689]" started-at="2022-11-06T21:13:13 EST" name="tearDown" finished-at="2022-11-06T21:13:13 EST" duration-ms="147" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.indexPageTest -->
    </test> <!-- IndexPageTest -->
    <test started-at="2022-11-06T21:13:02 EST" name="loginPageTest" finished-at="2022-11-06T21:13:38 EST" duration-ms="36232">
      <class name="testCases.loginPageTest">
        <test-method is-config="true" signature="loadConfig()[pri:0, instance:testCases.loginPageTest@15de0b3c]" started-at="2022-11-06T21:13:01 EST" name="loadConfig" finished-at="2022-11-06T21:13:02 EST" duration-ms="211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadConfig -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:testCases.loginPageTest@15de0b3c]" started-at="2022-11-06T21:13:02 EST" name="setup" finished-at="2022-11-06T21:13:14 EST" duration-ms="11834" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="doLoginTest()[pri:0, instance:testCases.loginPageTest@15de0b3c]" started-at="2022-11-06T21:13:14 EST" name="doLoginTest" description="This method tests valid login credentials" finished-at="2022-11-06T21:13:22 EST" duration-ms="8815" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLoginTest -->
        <test-method signature="doLogoutTest()[pri:1, instance:testCases.loginPageTest@15de0b3c]" started-at="2022-11-06T21:13:22 EST" name="doLogoutTest" description="This method tests logout is successful" finished-at="2022-11-06T21:13:34 EST" duration-ms="12093" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogoutTest -->
        <test-method signature="doLoginTestNegative()[pri:2, instance:testCases.loginPageTest@15de0b3c]" started-at="2022-11-06T21:13:34 EST" name="doLoginTestNegative" description="This method tests valid login credentials" finished-at="2022-11-06T21:13:37 EST" duration-ms="3024" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Homepage is not displayed after login...something went wrong!! expected object to not be null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Homepage is not displayed after login...something went wrong!! expected object to not be null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.assertNotNull(Assert.java:1319)
at testCases.loginPageTest.doLoginTestNegative(loginPageTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLoginTestNegative -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.loginPageTest@15de0b3c]" started-at="2022-11-06T21:13:38 EST" name="tearDown" finished-at="2022-11-06T21:13:38 EST" duration-ms="233" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:testCases.loginPageTest@15de0b3c]" started-at="2022-11-06T21:13:38 EST" name="afterSuite" finished-at="2022-11-06T21:13:38 EST" duration-ms="305" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testCases.loginPageTest -->
    </test> <!-- loginPageTest -->
  </suite> <!-- Suite -->
</testng-results>
